spec/Section 2 -- Language.md

```diff
@@ -184,7 +184,7 @@ represented in the shorthand form, which omits both the query keyword and
 operation name. Otherwise, if a GraphQL query document contains multiple
 operations, each operation must be named. When submitting a query document with
 multiple operations to a GraphQL service, the name of the desired operation to
-be executed must also be provided.
+be executed must also be provided. How the name is provided is up to the GraphQL implementation.
```

Comment:
This line is confusing - no where else do we specify that the information required to perform an execution needs to be provided by the implementation. For example, how the graphql document is provided is also defined by the implementation.

Perhaps to help alleviate the concern you mentioned, this should instead make it clear that the name should be provided in addition to the document. Suggestion:

When submitting a query document with multiple operations to a GraphQL service, the name of the desired operation to be executed must be provided *in addition to the document*.